docs/organization-integration/onpremise-analysis.md

```diff
@@ -91,7 +92,19 @@ We distinguish two types of environment variables:
 - Shared: these typically have the same value across different CI/CD projects for the same Sigrid 
   deployment. SIG recommends configuring these as variables managed by the CI/CD environment (often 
   called "secrets").
-- Non-shared: these typically differ across projects.
+- Non-shared: these typically differ between projects and should be configured individually in each pipeline.
+
+Using Shared Variables in Pipelines:
+- Using shared variables directly in the pipeline can be cumbersome. It's easier to either store all shared variables 
+  at a higher level in your CI/CD project hierarchy or use a template for them and include it in your pipeline. 
+  Make sure credentials are masked. Only the SYSTEM variable needs to be defined in the project pipeline itself, 
+  and the SIGRID_CI_TOKEN is best stored as a project-level variable.
+
+Example: Using a GitLab CI/CD template to store shared environment variables:
```

Comment:
While it would work, it's not ideal for secret values as the template resides in git in plain text. Does it need an extra hint to not put the SIGRID_CI_TOKEN in a template?